avro-builder/builder/src/main/java/com/linkedin/avroutil1/builder/SchemaBuilder.java

```diff
@@ -238,6 +244,11 @@ public static void main(String[] args) throws Exception {
       skipCodegenIfSchemaOnClasspath = Boolean.TRUE.equals(Boolean.parseBoolean(value));
     }
 
+    List<String> jsonPropsToIgnoreInCompare = new ArrayList<>();
+    if(options.has(jsonPropsToIgnoreInCompareOpt)) {
+      jsonPropsToIgnoreInCompare = options.valuesOf(jsonPropsToIgnoreInCompareOpt);
+    }
+
```

Comment:
nit: change to:

List<String> jsonPropsToIgnoreInCompare = options.has(jsonPropsToIgnoreInCompareOpt) ? options.valuesOf(jsonPropsToIgnoreInCompareOpt) : Collections.emptyList();